Drivers/seds/PioHwas/PioHwas.xml

```diff
@@ -0,0 +1,870 @@
+                                  <LongDescription>Assign mask to variable</LongDescription>
+                                  <VariableRef variableRef="pmcVar.PMC_PCxR0_PID16_PIOD_Mask" />
+                                </Assignment>
+                              </OnConditionTrue>
+                              <OnConditionFalse>
+                                <Conditional>
+                                  <LongDescription>Is port equal to port E?</LongDescription>
+                                  <Condition>
+                                    <Condition>
+                                      <FirstOperand variableRef="config.mPortConfig" />
+                                      <ComparisonOperator>equals</ComparisonOperator>
+                                      <Value value="PioHwas_Port_E" />
+                                    </Condition>
+                                  </Condition>
+                                  <OnConditionTrue>
+                                    <Assignment outputVariableRef="pinMaskVar">
+                                      <LongDescription>Assign mask to variable</LongDescription>
+                                      <VariableRef variableRef="pmcVar.PMC_PCxR0_PID17_PIOE_Mask" />
+                                    </Assignment>
+                                    <Assignment outputVariableRef="pinValueVar">
+                                      <LongDescription>Assign mask to variable</LongDescription>
+                                      <VariableRef variableRef="pmcVar.PMC_PCxR0_PID17_PIOE_Mask" />
+                                    </Assignment>
+                                  </OnConditionTrue>
+                                </Conditional>
+                              </OnConditionFalse>
+                            </Conditional>
+                          </OnConditionFalse>
+                        </Conditional>
+                      </OnConditionFalse>
+                    </Conditional>
+                  </OnConditionFalse>
+                </Conditional>
+                <SendCommandPrimitive shortDescription="Enable proper PIO PORT clock" interface="RawMemoryAccess" command="WriteWord">
+                  <ArgumentValue name="address">
+                    <VariableRef variableRef="destinationAddressVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="mask">
+                    <VariableRef variableRef="pinMaskVar" />
+                  </ArgumentValue>
+                  <ArgumentValue name="inputValue">
+                    <VariableRef variableRef="pinValueVar" />
+                  </ArgumentValue>
+                </SendCommandPrimitive>
+                <MathOperation outputVariableRef="pioVar.mPin">
+                  <LongDescription>Create pin mask</LongDescription>
+                  <Operator operator="pow" />
+                  <Value value="2" />
+                  <Value value="config.mPinConfig" />

```

Comment:
I think this should be a variable ref... please check if it works (it should!). If it doesn't, please report a bug on Jira and use the usual workaround...

Reply:
I will verify that.

Reply:
Verified. It does work. Fixed.